database/scripts/lib/buildfarm_tools.rb

```diff
@@ -41,6 +41,7 @@ def self.test_regressions_all(filter_known: false, include_reports: true)
     if filter_known
       known_errors = known_issues(status: 'open')
       known_errors.concat known_issues(status: 'disabled')
+      known_errors.concat known_issues(status: 'ignored')
```

Comment:
Maybe the word we are aiming for in those issues is "wontfix"? 

Ignored sounds like what happens with the issues that happen so rarely that are ignored, in this case, for those issues the tech team decided actively not to work on them. (also there's a wontfix label used already by the ros2 team that we can use to flag those issues in github)